platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerMBean.java

```diff
@@ -34,4 +35,11 @@
      * @throws MigrationException thrown if one or more Configurations couldn't be exported
      */
     Collection<MigrationWarning> export(@NotNull String exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
+     *
+     * @return Collection of type Describable
+     */
+    Collection<Describable> getInfo();
```

Comment:
We could keep things that way but it's probably not a good idea to keep them together. So what will most likely happen is that we'll have a class that implements the MBean interface, does any conversion required to comply with the MBean/Jolokia rules, and delegate to the actuall service behind the scenes. You can think of the MBean interface/class as an adaptor sitting on top of the actual service interface that does some kind of in/out conversion. We can talk about this some more tomorrow.